parquet-thrift/pom.xml

```diff
@@ -115,6 +115,11 @@
       <version>3.4</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.thrift</groupId>
+      <artifactId>libthrift</artifactId>
+      <version>${thrift.version}</version>
```

Comment:
Good to add this because there are direct dependencies on it. Usually it's provided by the application with thrift classes though, so could you change the scope to provided?